cfbs/module.py

```diff
@@ -0,0 +1,74 @@
+from collections import OrderedDict
+
+from cfbs.pretty import pretty
+
+
+class Module(OrderedDict):
```

Comment:
Please double-check the methods in the picture are OK for `Module` objects:
![image](https://user-images.githubusercontent.com/1044278/158221326-7f26ec51-4ca4-4ffb-a3af-607dabc72379.png)

Reply:
Also, what about `del module["name"]`?

Reply:
N/A